apps/docs/src/content/learn/basics/scheduling-tasks.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Scheduling Tasks
+category: Basics
+order: -750
+---
+
+In 3D apps and games, a lot of work is done in functions that run on every frame
+– from rotating an object continously around an axis to reacting to user input.
```

Comment:
```suggestion
In 3D apps and games, a lot of work is done in functions that run on every frame.
```
Recommend removing this example since a similar one is given at the end of this paragraph.